src/Bridge/Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -76,6 +78,10 @@ public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGenerator
         }
 
         $itemsPerPage = $resourceMetadata->getCollectionOperationAttribute($operationName, 'pagination_items_per_page', $this->itemsPerPage, true);
+        if ($this->enableGraphql) {
+            // @TODO Retrieve GraphQL 'first' argument using the request?
```

Comment:
Maybe can you create another `PaginationExtension` class specific to GraphQL if the handling is really different than this one.